melodic/distribution.yaml

```diff
@@ -2402,6 +2408,12 @@ repositories:
       url: https://github.com/yujinrobot/kobuki_core.git
       version: melodic
     status: maintained
+  kobuki_desktop:
+    source:
+      type: git
+      url: https://github.com/yujinrobot/kobuki_desktop.git
+      version: melodic
```

Comment:
Same issue with the branch name here. On which branch will melodic content be developed?

Reply:
:no_mouth:   let's imagine this has never happened... 
melodic branches pushed

Reply:
I retriggered it and it passed. Must have been a network glitch ;-)